rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+
+We propose that the metadata required to automatically identify the triplets (Table 1) is made optional and users are encouraged to submit these fields as it can help identify errors and also can in some cases provide important covariates for scientific analyses.
+
+**Table 1: Fields required for identification of fastq file correspondence**
+<table>
+  <tr>
+   <td><strong>Required Fields</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Justification</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Run Date and Time
+   </td>
+   <td>ISO Date and time at which the run was performed
+   </td>
+   <td>This is for the rare scenario where flowcells are re-used and ensures that each run is processed as if it were a separate flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Flowcell ID
+   </td>
+   <td>The Illumina identifier of the flowcell id
+   </td>
+   <td rowspan="3" >The unique combination of Flowcell ID, Lane and Barcode uniquely identifies any multiplexed library on an Illumina Flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Lane in Flowcell
+   </td>
+   <td>The lane in the flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Barcode of Library
+   </td>
+   <td>Barcode (or concatenation of the barcodes) of the library
+   </td>
+  </tr>
+</table>
+
+## Metadata Requirements for SS2 scRNA-seq datasets and provision for plate-based analyses
+The above metadata requirements are also compatible with SS2 analyses and can support plate based-analyses. Cells that have been sequenced more than once (a rare, but plausible scenario for SS2) can be identified by using a library identifier as presented in this RFC, currently under review.
+
+Furthermore, this approach can be extended to support plate-based processing. By ensuring that a plate field is provided for each cell, multiple cell bundles can be aggregated into a single plate-based workflow. A DAPS can contain one or more plates. Provision must be made for any cells not associated with a plate identifier, or otherwise, the plate identifier must be a compulsory field.
+
+## Pros and Cons of DAPS Approach
+
+Pros
+*   Avoids potentially very expensive run-away queries
+*   Ensures that analyses are triggered once with the appropriate datasets
+*   Is flexible and can accommodate all foreseeable data modalities
+*   Can immediately resolve processing challenges with 10X datasets
+*   Can provide a mechanism for moving to plate-based processing for SS2 in a seamless manner
+*   Does not preclude future automation of pipeline initiation when a mature system state is achieved
+*   Re-uses existing notification infrastructure
+*   Is permanently saved in the DSS and analyses are reproducible
+*   Provides a natural place to provide analysis-specific parameters
+
+Cons
+*   Analysis need to modify pipeline initiation scripts
+*   Initial implementation has human time cost of triggering the pipeline runs
```

Comment:
Why would this introduce human triggering of pipeline runs? when is a human needed in this process?

Reply:
this should be made a requirement on ingest to generate the signal.

Reply:
At least initially, Ingest could generate a signal that a whole project is ready for analysis when a submission completes. I think it's an open question how you'd signal any other kind of grouping like a dataset.